frontend/gql/apollo-config.js

```diff
@@ -1,9 +1,9 @@
 import { ApolloClient, InMemoryCache } from '@apollo/client';
 
- export const client = new ApolloClient({
-   uri: `https://api.baseql.com/airtable/graphql/${process.env.NEXT_PUBLIC_BASEQL_API_KEY}`,
-   cache: new InMemoryCache(),
-   headers: {
-     authorization: `Bearer ${process.env.NEXT_PUBLIC_BASEQL_BEARER_TOKEN}`,
-   },
- });
\ No newline at end of file
+export const client = new ApolloClient({
+    uri: `https://api.baseql.com/airtable/graphql/${process.env.NEXT_PUBLIC_BASEQL_API_KEY}`,
```

Comment:
Notice these indents? That's likely because our prettier configs are different. Do you mind adding a prettier config file that sets a rule for indentation spacing so that it's consistent for both of us?

Reply:
Changed config to have single indents instead of double